hoomd/mpcd/RotatedParallelPlateGeometry.h

```diff
@@ -32,12 +32,15 @@ class __attribute__((visibility("default"))) RotatedParallelPlateGeometry
     /*!
      * \param separation Distance between plates
      * \param angle Rotation angle (degrees)
-     * \param V Velocity of the wall
+     * \param speed Speed of the wall
      * \param no_slip Boundary condition at the wall (slip or no-slip)
      */
-    HOSTDEVICE RotatedParallelPlateGeometry(Scalar separation, Scalar angle, Scalar V, bool no_slip)
-        : m_H(Scalar(0.5) * separation), m_angle(angle * M_PI / 180.), m_V(V), m_no_slip(no_slip)
+    HOSTDEVICE
+    RotatedParallelPlateGeometry(Scalar separation, Scalar angle, Scalar speed, bool no_slip)
```

Comment:
I notice these should be merged into the same line. I will fix it!

Reply:
Yes, I would think those would go on the same line, unless `clang-format` is deciding they should be separated!